### DISCUSSION TITLE
[GHES] Adding CPU or memory resources on GCP

### DISCUSSION THREAD

**Discussion by shansrii** [community/community#131002] [Enterprise] [answered]
### Select Topic Area

General

### Body

Adding CPU or memory resources on GCP is missing on Github Documentation

[Adding CPU or memory resources](https://docs.github.com/en/enterprise-server@3.13/admin/monitoring-managing-and-updating-your-instance/updating-the-virtual-machine-and-physical-resources/increasing-cpu-or-memory-resources)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @shansrii, 

I reached out to the Docs team and I'm working with them to open an Issue so we can triage this. If you have any additional information or context I can add that would be incredibly helpful for the Docs team!

Thank you,
@queenofcorgis
